src/main/g8/README.md

```diff
@@ -137,6 +136,26 @@ Below are the libraries used to provide a broad starting base for this project.
 
 
 
+## [jmh](https://github.com/openjdk/jmh) - performance (micro)benchmarking ##
+
+`jmh` is a microbenching runner and harness (like a test framework, but for measuring and reporting rather than qualifying & asserting). 
+
+###### Why do we use it? ######
+
+- To gauge performance bottlenecks in isolation under a variety of circumstances
+- To tightly integrate with parts of your program and performance test them thoroughly
+- To win arguments with friends
+
+###### How do we use it? ######
+
+- SBT and [the sbt-jmh plugin](https://github.com/sbt/sbt-jmh) allow us to run via `sbt benchmarks/Jmh/run` or simply `sbt benchmark`
```

Comment:
I think it makes sense to add at least one of these as a real command that (by default) generates a results CSV, since...
1. That's how it is actually used, and...
2.  It illustrates that the jmh command can actually be parameterized, which is not mentioned elsewhere.